.travis.yml

```diff
@@ -18,7 +20,7 @@ after_success:
 jobs:
   include:
     - stage: Documentation
-      julia: 1.0
+      julia: 1.3
```

Comment:
```suggestion
      julia: 1
```

Reply:
tell me more...?

Reply:
Will build the documentation for the current release of julia.

In this case `1 => 1.3.1`

Reply:
Means less updates in the future

Reply:
cool. TIL